### DISCUSSION TITLE
Suggestions on markdown in jupyternotebook improvement

### DISCUSSION THREAD

**Discussion by MuRongKanKan** [microsoft/vscode-jupyter#9426] [New Ideas]
（From machine translation）
I've been learning to use the jupyter notebook in vscode recently. It's great, but I've also encountered some problems：
1. When editing markdown in Jupiter, you cannot insert pictures.
2. I hope the outline only lists the title, not all the markdown contents.
3. It seems that the # symbol of markdown cannot automatically generate serial number.

**Comment by DonJayamanne** [comment]
Thanks for the issues, I've created issues for each of the items and you can track the progress there.

* https://github.com/microsoft/vscode-jupyter/issues/9447
* https://github.com/microsoft/vscode-jupyter/issues/9448
* https://github.com/microsoft/vscode-jupyter/issues/9449

**Reply by DonJayamanne** [reply]
I'm going to close this discussion in favor of the issues.
